### ISSUE TITLE
Add screenshots depicting the app

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#30]
Screenshots are fun mate.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by anshumanv** [comment]
Ping @aashutoshrathi, please record a gif of using the app.

**Comment by deadbirdi** [comment]
I can do this one real quick @anshumanv

**Comment by deadbirdi** [comment]
Here we go:

[Evolutions Web App](https://i.imgur.com/NelTelr.gifv)

**Comment by anshumanv** [comment]
Awesome, please add it to projects readme, in some demo section :smile: 

Or do you think we need better backgrounds before shooting a demo? I just added them in haste for testing. If you're good with them do commit it. :+1:

**Comment by deadbirdi** [comment]
@anshumanv I actually am, personally, not a big fan of the changing backgrounds. But, if you're set on the backgrounds, I'd say lets get a few higher quality ones prior to adding it to the readme.

Actually, on that topic, are there plans to add more data beyond the sprites? I feel like it'd be dope to show the data below the Pokemon.

**Comment by anshumanv** [comment]
> I'd say lets get a few higher quality ones prior to adding it to the readme.

Makes sense, I'll update them shortly.

> Actually, on that topic, are there plans to add more data beyond the sprites? I feel like it'd be dope to show the data below the Pokemon.

Yeah since we already have the pokemon data coming from that monstrous JSON, what do you think of #28 ?

**Comment by deadbirdi** [comment]
> Yeah since we already have the pokemon data coming from that monstrous JSON, what do you think of #28 ?

I assigned it to myself. I'll get that one ready and then come back to this task.

**Comment by deadbirdi** [comment]
I went ahead and unassigned myself from the task as to give people participating in hacktoberfest an opportunity!
